bot/src/keeper/setup.ts

```diff
@@ -0,0 +1,47 @@
+import getSigner, { createSigner, setSigner } from 'auctions-core/src/signer';
+import {
+    KEEPER_COLLATERAL,
+    KEEPER_SURPLUS,
+    KEEPER_COLLATERAL_MINIMUM_NET_PROFIT_DAI,
+    KEEPER_SURPLUS_MINIMUM_NET_PROFIT_DAI,
+    KEEPER_WALLET_PRIVATE_KEY,
+} from '../variables';
+
+export const isSetupCompleted = (isSetupCompleted: boolean = false): boolean => {
+    return isSetupCompleted;
+};
+
+export const setupKeeper = async function (network: string) {
+    if (!KEEPER_WALLET_PRIVATE_KEY) {
+        console.warn('keeper: KEEPER_WALLET_PRIVATE_KEY variable is not set, keeper will not run');
+        return;
+    }
+    if (
+        !(KEEPER_COLLATERAL && !Number.isNaN(KEEPER_COLLATERAL_MINIMUM_NET_PROFIT_DAI)) ||
+        !(KEEPER_SURPLUS && !Number.isNaN(KEEPER_SURPLUS_MINIMUM_NET_PROFIT_DAI))
+    ) {
+        console.warn(
+            'keeper: boolean flags and profit variables are not set properly or are combined incorrectly, keeper will not run'
```

Comment:
Can we please reference the environment variables by name here?

This way a user does not need to go searching in the README what the boolean flags are, but can see what they are named just by reading your error message. (Might make sense to split the error message or make a longer one, up to you)